docusaurus/docs/create-a-plugin/extend-a-plugin/add-query-editor-help.md

```diff
@@ -41,7 +41,7 @@ Query editors support the addition of a help component to display examples of po
 
 1. Create a few examples of potential queries:
 
-   ```ts
+   ```tsx
```

Comment:
what is the criteria for when to use ts vs tsx?

Reply:
Whenever the code snippet contains jsx/markup the language should be set to tsx. Alternatively we set all to tsx and not use ts at all.

Reply:
gotcha thanks, definitely no change required for this PR. Just wondering if simpler for folks to then contribute to the docs if we're consistent with tsx for all.